iocage/cli/__init__.py

```diff
@@ -61,10 +61,16 @@
     exit(1)
 
 
-def print_events(generator: typing.Generator[IocageEvent, None, None]) -> None:
+def print_events(
+    generator: typing.Generator[typing.Union[IocageEvent, bool], None, None]
```

Comment:
what are the `None`s?
and is `bool` being a terminator a convention, or something you made up?